GameServer/Logic/Packets/Packets.cs

```diff
@@ -1181,6 +1181,22 @@ public SetCameraPosition(Champion champ, float x, float y, float z) : base(Packe
         }
     }
 
+    public class ShowHPAndName : BasePacket
+    {
+        public ShowHPAndName(Unit unit, bool show) : base(PacketCmdS2C.PKT_S2C_ShowHPAndName, unit.getNetId())
+        {
+            if (show)
```

Comment:
I'd rather do this section as follows:

```CS
var data = (byte)0xFC;
if(show)
{
    data = (byte)0xFD;
}

buffer.Write((byte)data);
buffer.Write((byte)0x00);
```

This is better for a few reasons:
* Less overall code
* More clear packet structure